tools/emscripten.py

```diff
@@ -893,6 +893,14 @@ def create_sending(metadata, library_symbols):
   return '{\n  ' + ',\n  '.join(elems) + '\n}'
 
 
+def can_use_await():
+  # In modularize mode we can use `await` since the factory function itself
+  # is marked as `async` and the generated code all lives inside that factory
+  # function.
+  # FIXME(https://github.com/emscripten-core/emscripten/issues/23158)
```

Comment:
The comment seems positive - "in mode X we can do Y, which is good" - so that there is a FIXME seems surprising. Perhaps
```suggestion
  # FIXME: support this with closure (https://github.com/emscripten-core/emscripten/issues/23158)
```